x-pack/plugins/uptime/public/components/monitor/ping_list/columns/ping_timestamp/step_image_popover.tsx

```diff
@@ -24,40 +26,108 @@ const StepImage = styled(EuiImage)`
     }
   }
 `;
+
+interface ScreenshotImageProps {
+  captionContent: string;
+  imageCaption: JSX.Element;
+}
+
+const DefaultImage: React.FC<ScreenshotImageProps & { url?: string }> = ({
+  captionContent,
+  imageCaption,
+  url,
+}) =>
+  url ? (
+    <StepImage
+      allowFullScreen={true}
+      alt={captionContent}
+      caption={imageCaption}
+      data-test-subj="pingTimestampImage"
+      hasShadow
+      url={url}
```

Comment:
I was focused very much on making sure composited images were working when I wrote this code, so the naming was secondary in my mind. I think I chose URL because, in the case of composited images, we're using `canvas` to create a data URI that we then provide to the `EuiImage`'s `url` field. We're likewise supplying a blob URI in the case of full-sized screenshots, so it seemed to fit.

I think I elected to use `url` instead of `src` because I seemed to recall that `EuiImage` treats them differently, but perhaps I was mistaken? Also we are already using the `url` prop in production today, and I didn't see any reason to change it.